torch/generic/Tensor.c

```diff
@@ -240,6 +240,22 @@ static int torch_Tensor_(clone)(lua_State *L)
   return 1;
 }
 
+static int torch_Tensor_(cloneOn)(lua_State *L)
```

Comment:
Do we still need the logic in `torch_Tensor_(clone)` (above) for auto-device mode? Maybe we should just assert that we're in manual mode (same for `cuda`), which would be more in line with the documentation.